app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,40 @@
+class TasksController < AuthenticatedController
+  include ProjectScoped
+
+  skip_before_action :set_project, if: -> { params[:project_id].blank? }
+  skip_before_action :set_nodes, if: -> { params[:project_id].blank? }
+
+  layout :set_layout
+  def index
+    @default_columns = ['Title', 'Methodology', 'Due Date', 'Assigned']
+
+    if params[:project_id].present?
+      cards = current_project.methodology_library.boards.map(&:cards)[0]
```

Comment:
Ditto. This can be moved to a `before_action`

```
def set_tasks
  @tasks ||= begin
    cards = current_user.cards

    if params[:project_id].present?
      cards.select { |card| card.project.id == current_project.id }
    else
      cards
    end
  end
end
```